src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const clonedObj = { ...state };
+  const result = [];
```

Comment:
[Use proper names as it is sain in checklist](https://github.com/mate-academy/js_stateful-clones/blob/master/checklist.md#:~:text=%5BNAMING%5D%20%2D%20use%20proper%20names%20for%20object%20copy)